.github/workflows/repo-stale.yaml

```diff
@@ -0,0 +1,31 @@
+name: Issue Stale Check
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
```

Comment:
Is this needed even if it doesn't close PRs?

Reply:
tbh I should clean that up entirely. this would only be effectife for the `{{ github.token }}` that gets injected into the CI run. 
Usually for in repo triggered workflows the token has full write access on all levels of the repo, unless the 'restrictive' mode is used, then you would be required to use the permission block...
Regardless, you are indeed correct, this would not be needed in any case if the workflow wont ever touch PRs.